HousingRepairsOnlineApi/Helpers/RepairDescriptionRequestToRepairDescriptionMapper.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using HousingRepairsOnlineApi.Domain;
+
+namespace HousingRepairsOnlineApi.Helpers;
+
+public class RepairDescriptionRequestToRepairDescriptionMapper : IRepairDescriptionRequestToRepairDescriptionMapper
+{
+    public RepairDescription Map(RepairDescriptionRequest repairRequest, string repairType)
+    {
+        var repairDescription = new RepairDescription { Text = repairRequest.Text };
+
+        switch (repairType)
+        {
+            case RepairType.Communal:
```

Comment:
It feels a little out of sorts to have switches on the repair type in helpers here. After a comment from @Bhavesh-Patel on my pull request i passed in a parameter instead of the repair type, e.g. 'includeLocationDescription'.